pkg/engine/loadbalancers.go

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
+
+import (
+	"fmt"
+
+	"github.com/Azure/aks-engine/pkg/api"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-08-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func CreateLoadBalancer(masterCount int) LoadBalancerARM {
+
+	var inboundNatRules []network.InboundNatRule
```

Comment:
```suggestion
	var inboundNatRules []network.InboundNATRule
```

Reply:
`network.InboundNATRule` is from Azure go SDK.